contents/architecture/RESTArchitecturalStyle.md

```diff
@@ -0,0 +1,88 @@
+
+As the combination of various types (video, audio, hyperlinks, text…) of information into one unit is becoming more common (e.g. in an online article teaching the reader how to use IntelliJ, there would be text describing the process, videos giving demonstrations and hyperlinks directing the reader to other articles), using an encompassing term to represent a unit of information makes sense as that is how the user organizes information. Moreover, this abstraction saves us the trouble of distinguishing the types or implementation of information inside this unit when thinking of the process of accessing and transmitting this unit of information.
+
+### Representational State
+
+Resource and the term *representational state* are closely related to the client-server model outlined as one of the rules of REST. Resource is the information stored at the server, whereas what are given/transfered to the clients when they access the resource are actually the "representations" of the resource at a specific point of time, i.e. the *representational state* of the resource. 
+
+### Representational State Transfer
+
+Transfering to the users the representation of the resource instead of the resources itself allows [dynamic binding](https://en.wikipedia.org/wiki/Late_binding) of the reference to a representation of the resource, and this enables the users to have access to and operate on this resource in the formats they want, such as JSON, HTML, XML etc. When the user wishes to make some changes to the resource, he/she can change the representation of this resource and send the representation back to server to update the resource. As such, the server is freed from managing different [application states](https://ruben.verborgh.org/phd/hypermedia/#the-statelessness-constraint-p-8) across requests. 
+
+The quote below from Roy Fielding outlines what the name entails:
+> The name “Representational State Transfer” is intended to evoke an image of how a well-designed Web application behaves: a network of web pages (a virtual state-machine), where the user progresses through the application by selecting links (state transitions), resulting in the next page (representing the next state of the application) being transferred to the user and rendered for their use.
+
+## REST defines what the Web is expected to be
```

Comment:
This is fine but it fits better with its peer headings if you can phrase it as a question `How does REST Define the Web?`, or as a topic phrase e.g., `REST as an Abstraction of the Web`. Right now, it is an assertion/statement. While it is fine to use assertions/statements as headings, it is odd to have only one such heading while all other headings are of a different form.